docs/RationaleFIRRTL.md

```diff
@@ -297,6 +297,16 @@ For a historical discussion of this issue and its development see:
 - [`llvm/circt#989`](https://github.com/llvm/circt/issues/989)
 - [`llvm/circt#992`](https://github.com/llvm/circt/pull/992)
 
+### `firrtl.bitcast`
+The bitcast operation represents a bitwise reinerpretation (cast) of a value. 
+It can be used to cast a vector or bundle type to an int type or vice-versa.
+The bitwidth of input and result type must be known and non-zero. It cannot
+be used for clock types or reset types 
```

Comment:
> It cannot be used for clock types or reset types 

Why not?  That seems like an arbitrary limitation, why not allow it?

Reply:
Added clock and reset types.